docs/guide_trivia_list_creation.rst

```diff
@@ -110,16 +110,67 @@ As you've added more questions, your file should look something like this:
     Who won the Premier League in 2015?:
     - Chelsea
     - chelsea f.c.
-    How much money is a US Olympic gold medalist awarded?:
-    - $25,000
-    - 25,000
-    - 25k
-    - 25000
-    - $25000
+    How much money is a US Olympic gold medallist awarded?:
+    - $37,500
+    - 37,500
+    - 37.5k
+    - 37500
+    - $37500
 
 You can keep adding questions until you are satisfied, and then you can upload and
 play your very own trivia! See :ref:`[p]triviaset custom <trivia-command-triviaset-custom>` for more information.
 
 Still stuck? Take a look at 
 `the core trivia lists <https://github.com/Cog-Creators/Red-DiscordBot/tree/V3/develop/redbot/cogs/trivia/data/lists>`_
 for reference.
+
+--------------------------
+Optional: Custom Overrides
+--------------------------
+
+Once you've got the hang of the question-answer format,
+you might want to add some custom overrides with the CONFIG key - in a similar way to the AUTHOR key.
+These will override the settings set with :ref:`[p]triviaset <trivia-command-triviaset>`.
+For example, with a trivia list which has questions that are quick to answer you could decrease the time limit
+and require a higher score to win.
+
+Here are all the overrides available:
+
+.. code-block:: yaml
+
+    CONFIG:
+        bot_plays: true or false  # bot gains points if no one answers correctly
+        delay: positive number  # answer time limit (seconds), must be greater than or equal to 4
+        timeout: positive number  # timeout for no responses (seconds), must be greater than delay
+        max_score: positive integer  # points required to win
+        reveal_answer: true or false  # reveal answer on timeout
+        payout_multiplier: non-negative number  # payout multiplier
+        use_spoilers: true or false  # use spoilers in answers
+
+So, your final file might look something like this:
+
+.. code-block:: yaml
+
+    AUTHOR: Red
+    CONFIG:
+        bot_plays: false
+        use_spoilers: true
+        delay: 20
```

Comment:
I suggest adding a couple more numeric examples to show what can be passed (hopefully better shows the difference between an integer and a number):
```suggestion
        delay: 20
        payout_multiplier: 0.5
        max_score: 20
```